packages/expo-file-system/android/src/main/java/expo/modules/filesystem/next/FileSystemNextModule.kt

```diff
@@ -16,11 +19,19 @@ import java.io.FileOutputStream
 import java.net.URI
 
 class FileSystemNextModule : Module() {
+  private val context: Context
+    get() = appContext.reactContext ?: throw Exceptions.AppContextLost()
 
   @OptIn(EitherType::class)
   override fun definition() = ModuleDefinition {
     Name("FileSystemNext")
 
+    Constants(
+      "documentDirectory" to Uri.fromFile(context.filesDir).toString() + "/",
+      "cacheDirectory" to Uri.fromFile(context.cacheDir).toString() + "/",
+      "bundleDirectory" to "asset:///"
+    )
```

Comment:
I would use properties instead of constants. cc @tsapeta

Reply:
In this specific case I'm ok with both, but in general if obtaining the "constant" is expensive it is better to use properties (all constants are evaluated on module initialization)

Reply:
Makes sense in general, but constants are fine for this case imo